resources/assets/components/pages/AccountPage/Subscriptions/CancelEmailSubscription.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+
+import Spinner from '../../../artifacts/Spinner/Spinner';
+
+const EMAIL_SUBSCRIPTION_STATUS = gql`
+  query EmailSubscriptionStatus($userId: String!) {
+    user(id: $userId) {
+      id
+      emailSubscriptionStatus
+    }
+  }
+`;
+const EMAIL_SUBSCRIPTION_STATUS_MUTATION = gql`
+  mutation EmailSubscriptionStatus(
+    $userId: String!
+    $emailSubscriptionStatus: Boolean!
+  ) {
+    updateEmailSubscriptionStatus(
+      id: $userId
+      emailSubscriptionStatus: $emailSubscriptionStatus
+    ) {
+      id
+      emailSubscriptionStatus
+    }
+  }
+`;
+const CancelEmailSubscription = props => {
+  const userId = props.user.id;
+  const options = { variables: { userId } };
+
+  const { data, loading, error } = useQuery(EMAIL_SUBSCRIPTION_STATUS, options);
+  const [updateEmailSubscriptionStatus] = useMutation(
+    EMAIL_SUBSCRIPTION_STATUS_MUTATION,
+    options,
+  );
+
+  if (error) {
+    return <p>Something went wrong!</p>;
+  }
+
+  if (loading) {
+    return Spinner;
+  }
+  return (
+    <div>
+      {data.user.emailSubscriptionStatus ? (
+        <p>
+          Need a break?
+          <button
+            className="px-1 text-blue-500 pb-4"
+            type="button"
+            onClick={() =>
+              updateEmailSubscriptionStatus({
+                variables: {
+                  userId,
+                  emailSubscriptionStatus: false,
+                },
```

Comment:
And oops, https://github.com/DoSomething/phoenix-next/pull/2201#discussion_r446295352 actually applies here! For the mutation:
```suggestion
              updateEmailSubscriptionStatus({
                variables: {
                  userId,
                  emailSubscriptionStatus: false,
                },
                refetchQueries: ['EmailSubscriptionsQuery'],
```

Reply:
@mendelB should I also add refetchQueries to emailSubscriptionTopics? There's an error with re-subscribing to individual topics after globally unsubscribing

Reply:
I think we will need to refetch the `EmailSubscriptionsQuery` so that we can re-display the cancel subscriptions link once we fix the bug described [here](https://dosomething.slack.com/archives/CUQMU4Q6B/p1593191113010300?thread_ts=1593187616.006000&cid=CUQMU4Q6B). But I'm not quite sure what this error is about 👀

Reply:
Oh I see, this is all down here! I'm surprised this doesn't update the other query's results without this argument. 🤔 No need to block merging, but there should be a way to avoid explicitly tying these two together...